package ExceptionHandlingInJava;

import java.util.Scanner;

public class FinallyRunner {

	public static void main(String[] args) {
		try {
			
			Scanner scan = new Scanner(System.in);
			
			int[] numbers = {12,2,3,45};
			
			int number = numbers[6];
			
			System.out.println("Before Scanner Close");
			scan.close();
			
			
			
		}
		catch(Exception e) {
			e.printStackTrace();
			
		}

	

	}

}
